app/plot_results.R

```diff
@@ -45,14 +45,17 @@ idx = 100
 
 f_cck_ii = df_beta$f_cck[idx]
 d_ii = df_beta$d[idx]
+d_lower_ii = df_beta$d_lower[idx]
+a_to_d_ratio_ii = df_beta$a_to_d_ratio[idx]
 hvar = "chi1"
 hfacet = "load_comb"
 vfacet = "rho"
 color = "chi2"
 
 g = beta_ggplot(
   df_beta=df_beta, hvar=hvar, hfacet=hfacet, vfacet=vfacet,
-  color=color, f_cck_ii=f_cck_ii, d_ii=d_ii)
+  color=color, f_cck_ii=f_cck_ii, d_ii=d_ii, d_lower_ii=d_lower_ii,
```

Comment:
If the dashboard only corresponds to the calibrated EN1992-1-1 shear resistance model (see Readme.md), there is no need to include d_lower and a/d. On the other hand, if we want to change it later, their presence is convenient.

Reply:
It is resistance model agnostic, works with all the outputs that are produced by the calibration code, including the MC2010 model. Readme.md only tells that the default displayed data is EN1992-1-1, you can visualize other results as well, that's why we have the upload data features. (The currently displayed results are already about 5MB so I would not add more results to the repository, however we could share csv files corresponding to different runs in a public storage).